custom/m4change/models.py

```diff
@@ -52,7 +52,6 @@ class BaseM4ChangeCaseFluff(fluff.IndicatorDocument):
     document_class = XFormInstance
     document_filter = ORFilter(_get_all_m4change_forms())
     domains = M4CHANGE_DOMAINS
-    kafka_topic = topics.CASE
```

Comment:
pretty egregious error here. I guess i'll blame the class name to make myself feel better.